tests/provision/bootc/test.sh

```diff
@@ -11,10 +11,20 @@ rlJournalStart
         # in the podman machine mount
         rlRun "mkdir -p /var/tmp/tmt"
         rlRun "run=\$(mktemp -d --tmpdir=/var/tmp/tmt)" 0 "Create run directory"
+        rlRun "dry_run=\$(mktemp -d --tmpdir=/var/tmp/tmt)" 0 "Create dry run directory"
+        rlRun "dry_run_image='quay.io/fedora/fedora-bootc:41'" 0 "Set dry run container image"
```

Comment:
As mentioned in https://github.com/teemtee/tmt/pull/3751#pullrequestreview-2868034139, the image name should not really be hardcoded in the test like this, and the upstream image shouldn't be used directly either.

Since we do not have any `bootc` image ready for tests, we should get one, `containers/fedora/latest/bootc/Containerfile` maybe? With a corresponding additions to `Makefile`, you would then add `build_container_image "fedora/latest/bootc\:latest"` to the setup phase, and then use `$TEST_IMAGE_PREFIX/fedora/latest/bootc:latest` in your test, instead of the hardcoded versioned image name.